boards/arm/blt_plane/blt_plane.dts

```diff
@@ -0,0 +1,300 @@
+/*
+ * Copyright (c) 2025 Launch Initiative
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/f4/stm32f446Xe.dtsi>
+#include <st/f4/stm32f446r(c-e)tx-pinctrl.dtsi>
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+
+/ {
+	model = "Power Module";
+	compatible = "st,stm32f446xx";
+
+	chosen {
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,console = &uart5;
+		zephyr,shell-uart = &uart5;
+
+		storage = &w25q128;
+		logfs = &lfs1;
+	};
+
+	aliases {
+		inabatt = &ina0;
+		ina3v3 = &ina1;
+		ina5v0 = &ina2;
+		vin = &adc1;
+		rtc = &rv8263;
+	};
+
+	leds: leds {
+		compatible = "gpio-leds";
+
+		led_0: led_0 {
+			gpios = <&gpiob 12 GPIO_ACTIVE_HIGH>;
+		};
+
+		led_1: led_1 {
+			gpios = <&gpiob 13 GPIO_ACTIVE_HIGH>;
+		};
+
+		led_2: led_2 {
+			gpios = <&gpiob 14 GPIO_ACTIVE_HIGH>;
+		};
+
+		led_3: led_3 {
+			gpios = <&gpiob 15 GPIO_ACTIVE_HIGH>;
+		};
+
+		wiznet_led: wiznet_led {
+			gpios = <&gpioc 7 GPIO_ACTIVE_HIGH>;
+		};
+
+		rtc_en_clk_out: rtc_en_clk_out {
+			gpios = <&gpiob 5 GPIO_ACTIVE_LOW>; // this defaults to enabling CLKOUT
+			label = "RV8263 Enable Clock Out";
+		};
+	};
+
+	fstab {
```

Comment:
ive become enlightened on idiomatic zephyr (i think) and hot take this shouldnt go in the board dts. It should go in the app_folder/boards/blt.overlay

If we needa completely different mission (unlikely), or phase out littlefs (more likely) we'd have to change the board dts when thats an application level issue

unfortuanetely im only realizing this after all the boards ive brought up other than sw board 💀

Reply:
How about i do this in a separate PR for all the boards